python/pyspark/testing/connectutils.py

```diff
@@ -57,5 +57,7 @@ def __getattr__(self, item: str) -> Any:
 class PlanOnlyTestFixture(unittest.TestCase):
     @classmethod
     def setUpClass(cls: Any) -> None:
-        cls.connect = MockRemoteSession()
+        cls._connect = MockRemoteSession()
         cls.tbl_name = f"tbl{uuid.uuid4()}".replace("-", "")
+
+    connect: "MockRemoteSession"
```

Comment:
Let's put this type hint under `class PlanOnlyTestFixture` (see https://github.com/apache/spark/blob/master/python/pyspark/sql/tests/test_connect_basic.py#L34)

Reply:
ah sure looks like we want to keep with the existing styple.